activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/model/VariableDefinition.java

```diff
@@ -0,0 +1,61 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.spring.process.model;
+
+public class VariableDefinition {
+
+    private String id;
+    private String name;
+    private String description;
+    private String type;
+    private boolean required;
+    private Object value;
```

Comment:
@ryandawsonuk @balsarori this should be defaultValue right? let's make sure that we don't confuse people.

Reply:
Also.. does this push us to deserialise the default value?